modules/core/src/test/java/org/apache/ignite/spi/communication/tcp/IgniteTcpCommunicationHandshakeWaitTest.java

```diff
@@ -108,6 +108,16 @@ public void testHandshakeOnNodeJoining() throws Exception {
         fut.get();
     }
 
+    /** */
+    private long expectedTimeout() {
```

Comment:
Why ExponentialBackoffTimeoutStrategy is not used here ?

Reply:
Cause we are checking correctness of actual max backoff timeout here in test

Reply:
Max wait should be calculated using newly introduced maxWaitTime method in TimeoutStragefy interface.